### PR TITLE
PMM-2244 Show client version in status.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm-admin?pullRequest=105) <br/>All committers have signed the CLA.

**Comment by BupycHuk** [comment]
@JiriCtvrtka travis CI build are failed, please fix it

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Still failing

**Comment by JiriCtvrtka** [comment]
@BupycHuk Yes I know, test passed on local because port 7777 was avalaible. I will change it.

**Comment by JiriCtvrtka** [comment]
I added split to version string to match task description. Also I've looked at the Parse method from version package as well, but I'd have to join and stringify again. That's why this solution seemed to be better from my point of view.

File: agentlocal/agentlocal_test.go
Comment (BupycHuk) [inline agentlocal/agentlocal_test.go]: we don't write test this way. Please use assert or require.|
If you want to write test for the call to pmm-agent it's better to mock the answer.

File: agentlocal/agentlocal.go
Comment (BupycHuk) [inline agentlocal/agentlocal.go]: I would suggest to not add this field to this struct. It can be added as separate field in status.go

File: commands/list_test.go
Comment (BupycHuk) [inline commands/list_test.go]: What's the reason to add fields which are not required in current test?

File: commands/list_test.go
Comment (BupycHuk) [inline commands/list_test.go]: The same comment as in previous test file

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: agentlocal/agentlocal.go
Comment (JiriCtvrtka) [reply agentlocal/agentlocal.go]: Moved into status.go.

File: commands/list_test.go
Comment (JiriCtvrtka) [reply commands/list_test.go]: You are right. I forgot delete it.

File: agentlocal/agentlocal_test.go
Comment (JiriCtvrtka) [reply agentlocal/agentlocal_test.go]: Assert used in all cases. It was generated and I forgot that we use errors package for tests. Now I  think about this test, if I mock it then it actually unnecessary test I think. I added it, because codecov.io wanted to cover it. I will try invert test which make sense even with mock.

Comment (JiriCtvrtka) [reply agentlocal/agentlocal_test.go]: @BupycHuk  I removed test and coverage is OK, even without it. So should I let as it is? Because I think with mock it is unnecessary test. Other issues should be fixed.

File: commands/status.go
Comment (BupycHuk) [inline commands/status.go]: please leave as is, no need to split it

Comment (JiriCtvrtka) [reply commands/status.go]: Sure, just few minutes. I split it, because in description it was without -xxxx.

Comment (BupycHuk) [reply commands/status.go]: yeah, -xxxx visible only in dev builds

Comment (JiriCtvrtka) [reply commands/status.go]: Ok done.

**Review by BupycHuk — APPROVED** [review]
Let's create FB and move it to In QA status.

**Comment by JiriCtvrtka** [comment]
@BupycHuk I cannot merge it because I still dont have write access here. Please can you do that? Also I am not sure about Percona-Lab/pmm-submodules.

File: commands/status.go
Comment (askomorokhov) [inline commands/status.go]: Strictly speaking, it is a version of pmm-admin but not pmm-agent.

```
[root@fa40586bdea1 /]# pmm-agent --version
ProjectName: pmm-agent
Version: 2.6.0
PMMVersion: 2.6.0
Timestamp: 2020-05-05 12:50:58 (UTC)
FullCommit: d66ebd798de738d9c1711ea02e57fb0cec5b0962
[root@fa40586bdea1 /]# pmm-admin --version
ProjectName: pmm-admin
Version: 2.5.0-7-g4ab1e51-dirty
PMMVersion: 2.5.0-7-g4ab1e51-dirty
Timestamp: 2020-05-06 09:50:59 (UTC)
FullCommit: 4ab1e51f86d9e19c0173af409bd8c7b40fa9d784
Branch: PMM-5776-inv-ex-service
```

My suggestion to add a new section:
```
PMM Admin: 
  Version   : {{ .PMMVersion }}
```

Otherwise, I guess we need to extend the Status method to return the version of pmm-agent. https://github.com/percona/pmm-agent/blob/master/agentlocal/agent_local.go#L127

File: commands/status.go
Comment (askomorokhov) [inline commands/status.go]: Maybe we need to add a struct field tag to be consistent.

```suggestion
	PMMVersion     string `json:"pmm_admin_version"`
```

**Review by askomorokhov — CHANGES_REQUESTED** [review]

**Review by askomorokhov — APPROVED** [review]

File: commands/status.go
Comment (AlekSi) [inline commands/status.go]: Please use `pmm-admin`, `pmm-agent`, `PMM Client`, `PMM Server` everywhere.
It should be `pmm-admin` there, and `pmm-agent` above.

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: commands/status.go
Comment (BupycHuk) [inline commands/status.go]: let's remove it until we have pmm-agent version from API.

File: commands/status_test.go
Comment (BupycHuk) [inline commands/status_test.go:58]: pmm-admin version should be visible even if pmm-agent is not connected.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**
